common/config/config.go

```diff
@@ -2,23 +2,34 @@ package config
 
 import (
 	"github.com/HackIllinois/api/common/configloader"
+	"github.com/HackIllinois/api/common/errors"
 	"os"
+	"strings"
 )
 
 var IS_PRODUCTION bool
+var DEBUG_MODE bool
 
 func init() {
 	cfg_loader, err := configloader.Load(os.Getenv("HI_CONFIG"))
 
 	if err != nil {
-		panic(err)
+		panic(errors.InternalError(err.Error(), "Could not load from config file."))
```

Comment:
`panic(err)` is correct here since these errors don't go through the errors middleware.